### PR TITLE
add exercise 14 files

### PR DESCRIPTION
**Pull request by barrantes-sofiaanne**
Please review my file. Thank you!

### CODE REVIEW

File: barrantes/countries.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Countries of the World</title>
+    <link rel="stylesheet" href="countries.css">
+</head>
+<body>
+    <div class="container">
+        <h1>Countries of the World</h1>
+        <input type="text" id="countryInput" placeholder="Enter country name..">
```
Comment (ElizaAlimasa) [inline barrantes/countries.html]: IDs should be written in snake case. All lower case separated by underscore. For reference, please click on the link provided below to access further information.

```suggestion
        <input type="text" id="country_input" placeholder="Enter country name..">
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)

File: barrantes/countries.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        <button onclick="searchCountry()">Search</button>
+        <div class="country-details" id="countryDetails"></div>
```
Comment (ElizaAlimasa) [inline barrantes/countries.html]: IDs should be written in snake case. All lower case separated by underscore. For reference, please click on the link provided below to access further information.

```suggestion
        <div class="country-details" id="country_details"></div>
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)

File: barrantes/countries.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        <div class="same-region-countries" id="sameRegionCountries"></div>
```
Comment (ElizaAlimasa) [inline barrantes/countries.html]: IDs should be written in snake case. All lower case separated by underscore. For reference, please click on the link provided below to access further information.

```suggestion
        <div class="same-region-countries" id="same_region_countries"></div>
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)

File: barrantes/index.html [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+    <div class="contact">
+        <h2>Contact Me</h2>
+        <form>
+            <div class="email">
+                <label for="email">Email:</label>
+                <input type="email" id="email" name="email" />
+            </div>
+            <div class="name-contact">
+                <label for="Name">Full Name:</label>
+                <input type="text" id="name-contact" name="Name" />
```
Comment (ElizaAlimasa) [inline barrantes/index.html]: IDs should be written in snake case. All lower case separated by underscore. For reference, please click on the link provided below to access further information.

```suggestion
                <input type="text" id="name_contact" name="Name" />
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)

File: barrantes/index.html [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+                            <label for="fname">Name: </label>
+                            <input type="text" name="fname" id="fname">
+                        </div>
+                        <div class="com">
+                            <label for="comment">Comment: </label>
+                            <textarea id="comment" name="comment" rows="5" 
+                            cols="33"></textarea>
+                        </div>
+                        <div class="com-b">
+                            <button disabled type="submit" id="com-b">Submit
```
Comment (ElizaAlimasa) [inline barrantes/index.html]: IDs should be written in snake case. All lower case separated by underscore. For reference, please click on the link provided below to access further information.

```suggestion
                            <button disabled type="submit" id="com_b">Submit
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)

**Review by ElizaAlimasa — CHANGES_REQUESTED** [review]
Kindly review the suggested changes in your files and make sure to update your js and css file.

**Review by ElizaAlimasa — APPROVED** [review]
Everything looks good to me. You can merge it now

**Review by ElizaAlimasa — APPROVED** [review]
Great! Merge it now

**Comment by barrantes-sofiaanne** [comment]
Thank you!

**Pull request merged**